src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureTwo implements AreaOfFigure, InfoFigure {
+    //прямоугольный треугольник
+    private double height; // высота
+
+    public RightTriangle(String color, double firsSide, double height) {
```

Comment:
There is a typo in the constructor parameter name `firsSide`. It should be `firstSide` to match the class property name.